multidep.json

```diff
@@ -0,0 +1,6 @@
+{
+  "path": "multidep",
+  "versions": {
+    "aframe": ["0.2.0"]
```

Comment:
😢 it's sad we don't have `0.1.0` docs. at the very least, can we have `/docs/0.1.0/guide/` not 404?